### PR TITLE
FR-1963 Amend document display in Contested

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FR-1963

### Change description ###
- Re-created all collections to reflect document display (10 for each Applicant and Respondent) 
- Created filters for all collections

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsServiceTest.java [hunk 1]
```diff
@@ -59,7 +79,8 @@ public void setUp() {
 
     @Test
     public void applicantCorrespondenceDocumentsFiltered() {
-        uploadDocumentList.add(createContestedUploadDocumentItem("Letter from Applicant", "applicant", "no", null));
+        when(featureToggleService.isRespondentJourneyEnabled()).thenReturn(false);
```
Comment (ravinderpal) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsServiceTest.java:82]: Added feature toggle for old tests.

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsServiceTest.java [hunk 2]
```diff
@@ -69,11 +90,12 @@ public void applicantCorrespondenceDocumentsFiltered() {
 
     @Test
     public void applicantFrFormsDocumentsFiltered() {
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form B", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Applicant - Form E", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form F", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form G", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form H", "applicant", "no", null));
+        when(featureToggleService.isRespondentJourneyEnabled()).thenReturn(false);
+        uploadDocumentList.add(createContestedUploadDocumentItem("Form B", "applicant", "no", null, null));
```
Comment (ravinderpal) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsServiceTest.java:94]: Added new 'null' parameter to all old tests to reflect the addition of "hearingDetails' text field

**Review by shaunganley — APPROVED** [review]
great work

**Review by jakub-kozlowski — APPROVED** [review]

**Comment by ravinderpal** [comment]
Temporary closing PR to release resources

**Pull request merged**